src/components/attendee/VideoComms/internal/useMute.ts

```diff
@@ -0,0 +1,16 @@
+import { useCallback, useState } from "react";
+
+export const useMute = () => {
```

Comment:
[NOTE] `useMute` is basically `useShowHide`. Both just have too specific names for something that generically be named `useToggle`.